src/resolvers/user/mutation/user.js

```diff
@@ -1,7 +1,5 @@
 export default {
   Mutation: {
-    createUsers: async (obj, args, context, requestInfo) => {
-      console.log(obj, args, context, requestInfo);
-    },
+    createUsers: async (obj, args, context, requestInfo) => {},
```

Comment:
Muito clean, falta um console.log


```suggestion
    createUsers: async (obj, args, context, requestInfo) => {
        console.log("Debugando");
    },
```